readthedocs/search/parsers.py

```diff
@@ -307,10 +308,42 @@ def parse(self, page):
             'domain_data': {},
         }
         """
-        raise NotImplementedError
+        try:
+            content = self._get_page_content(page)
+            if content:
+                return self._process_content(page, content)
+        except Exception as e:
+            log.info("Failed to index page.", path=page, exc_info=True)
```

Comment:
Do we want to log a higher level for now to test this in production? I imagine we might hit some valid exceptions, but not sure.

Reply:
Do you mean use log.warning?

Reply:
Yes.